tools/tpm2_quote.c

```diff
@@ -212,6 +214,81 @@ UINT16  calcSizeofTPMT_SIGNATURE( TPMT_SIGNATURE *sig )
     return size > sizeof(*sig) ? sizeof(*sig) : size;
 }
 
+FILE* open_file(const char *path)
+{
```

Comment:
Even though not all files comply or even come close to the coding standard, new hunks should comply, please see: misc/coding_standard_c.txt